zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -57,14 +57,15 @@
   FormType formType;
   boolean initialized;
   private Map<String, String> env;
-
   private int connectTimeout;
+  private int maxPoolSize;
 
   public RemoteInterpreter(Properties property,
                            String className,
                            String interpreterRunner,
                            String interpreterPath,
                            int connectTimeout,
+                           int maxPoolSize,
```

Comment:
You'll also need to add `maxPoolSize` and initialize the value in the other constructor around line 86